base/serialize.jl

```diff
@@ -437,7 +437,9 @@ function serialize(s::AbstractSerializer, t::Task)
 end
 
 function serialize(s::AbstractSerializer, g::GlobalRef)
-    if g.mod === Main && isdefined(g.mod, g.name) && isconst(g.mod, g.name)
+    if (g.mod === __deserialized_types__ ) ||
+        (g.mod === Main && isdefined(g.mod, g.name) && isconst(g.mod, g.name))
```

Comment:
Wastes space, and the line continuation looks like it's a normal statement within the block at a glance. Distinguishing it with a different indentation level helps a lot and is done in many other places.